packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -237,6 +237,24 @@ function GlobalStylesBlockLink() {
 	}, [ selectedBlockClientId, selectedBlockName, blockHasGlobalStyles ] );
 }
 
+function GlobalStylesRevisionLink() {
```

Comment:
Is common pattern, e.g., a "hook component"? 😄 

I think I see why it's needed - calling the hook in `GlobalStylesUI()` doesn't seem to work. 

Maybe return `null` for the purists? I don't know 🤷

Reply:
I originally wrote it as a hook, and then realised it needed to be a component so that it can be executed from within the provider. I then copied the component directly above this one, and added a `return null`, but then removed the `return null` when I saw that the above component doesn't have it.

I personally slightly prefer the explicit `return null`, as it's clearer that it really is a component that doesn't render anything. Happy to add it back in.

Reply:
Thanks for the explanation! Either way is fine I reckon. It's working as intended!